plugins/protocolbuffers/pyi/v26.0/buf.plugin.yaml

```diff
@@ -0,0 +1,18 @@
+version: v1
+name: buf.build/protocolbuffers/pyi
+plugin_version: v26.0
+source_url: https://github.com/protocolbuffers/protobuf
+description: Base types for Python Stubs. Generates stub files for message and enum types.
+output_languages:
+  - python
+spdx_license_id: BSD-3-Clause
+license_url: https://github.com/protocolbuffers/protobuf/blob/v26.0/LICENSE
+registry:
+  python:
+    package_type: "stub-only"
+    # https://github.com/protocolbuffers/protobuf/blob/v26.0/python/dist/setup.py#L78
+    requires_python: ">=3.8"
+    deps:
+      # https://pypi.org/project/protobuf/
+      - "protobuf"
```

Comment:
@stefanvanburen - In light of https://protobuf.dev/support/cross-version-runtime-guarantee/#major, I wonder if we don't want to be more constrained here. The upstream package bumped from major version 4 to 5 with this release.

Reply:
yeah, was just looking at this. I think you're right. Debating between a strict `==` specifier or using the `~=` compatible specifier ([docs](https://packaging.python.org/en/latest/specifications/version-specifiers/#id5)); thoughts based on what we use elsewhere?

Reply:
i.e. I think we should either pin directly to `==5.26.0`, or do something like `~=5.26` (so `5.26.0` or `5.26.1` are valid, but not `5.27`) or `~=5` for `5.*`.

Reply:
I think using the compatible specifier would be good to start. On the Java side we pull in an exact version but users can pull in a later version when consuming the artifacts.

Reply:
yeah. Maybe `~=5` to match anything within the major version, since in theory that's where we'd get in trouble with incompatibilities?

Reply:
Settled on `~= 5.26` for now which should be a best effort to pull in a compatible version of the runtime but not be too constrained.